Bio/phenotype/phen_micro.py

```diff
@@ -112,18 +109,29 @@ class PlateRecord(object):
     H01
     H02
     H03
-    ...
```

Comment:
Why fill out the full list? You could list 1, 2, 3, ..., 12 if you wanted to be a little more explicit.